azure-spring-cloud-maven-plugin/src/main/java/com/microsoft/azure/maven/springcloud/DemoMojo.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.springcloud;
+
+import com.microsoft.azure.common.exceptions.AzureExecutionException;
+import com.microsoft.azure.management.appplatform.v2020_07_01.implementation.AppPlatformManager;
+import com.microsoft.azure.maven.utils.MavenArtifactUtils;
+import com.microsoft.azure.toolkit.lib.springcloud.AzureSpringCloud;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudApp;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudCluster;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudDeployment;
+import com.microsoft.azure.toolkit.lib.springcloud.config.SpringCloudAppConfig;
+import com.microsoft.azure.toolkit.lib.springcloud.config.SpringCloudDeploymentConfig;
+import com.microsoft.azure.toolkit.lib.springcloud.model.ScaleSettings;
+import com.microsoft.azure.toolkit.lib.springcloud.model.SpringCloudRuntimeVersion;
+import org.apache.maven.plugin.MojoExecutionException;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Optional;
+
+public class DemoMojo {
+
+    private static final int GET_STATUS_TIMEOUT = 180;
+
+    protected void doExecute() throws MojoExecutionException, AzureExecutionException {
+        // Init spring clients, and prompt users to confirm
+        final SpringCloudAppConfig appConfig = null;
+        final File artifact = MavenArtifactUtils.getArtifactFromTargetFolder(null);
+        final boolean enableDisk = appConfig.getDeployment() != null && appConfig.getDeployment().isEnablePersistentStorage();
```

Comment:
convert isEnablePersistentStorage(Boolean) to boolean is dangerous

Reply:
please check `SpringCloudDeploymentConfig.isEnablePersistentStorage`